Paco-Server/src/com/google/sampling/experiential/server/ACLHelper.java

```diff
@@ -41,6 +42,11 @@ public static String getModifiedQueryBasedOnACL(Select selectSql, String loggedI
     if (userSpecifiedExpIdValues.size() == 0) {
       throw new Exception(ErrorMessages.EXPERIMENT_ID_CLAUSE_EXCEPTION.getDescription());
     }
+    
+    // is the user only a participant, has never been an admin
```

Comment:
Just to clarify the language, it seems like it should say "is the user only a participant". The "has never been an admin" phrase seems confusing because we don't keep a history of if they have ever been an admin - it only matters if they are presently an admin.